sdks/java/io/elasticsearch/src/test/contrib/create_elk_container.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/sh
+################################################################################
+#
+#    Licensed to the Apache Software Foundation (ASF) under one or more
+#    contributor license agreements.  See the NOTICE file distributed with
+#    this work for additional information regarding copyright ownership.
+#    The ASF licenses this file to You under the Apache License, Version 2.0
+#    (the "License"); you may not use this file except in compliance with
+#    the License.  You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+#
+################################################################################
+
+#Create an ELK (Elasticsearch Logstash Kibana) container for ES v2.4 and compatible Logstash and Kibana versions,
+#bind then on host ports, allow shell access to container and mount current directory on /home/$USER inside the container
+
+docker create -p 5601:5601 -p 9200:9200 -p 5044:5044 -p 5000:5000 -p 9300:9300 -it -v $(pwd):/home/$USER/ --name elk-2.4  sebp/elk:es240_l240_k460
+
+
+#Create an ELK (Elasticsearch Logstash Kibana) container for ES v5.0 and compatible Logstash and Kibana versions,
+#bind then on host ports, allow shell access to container and mount current directory on /home/$USER inside the container
+
+docker create -p 5601:5601 -p 9200:9200 -p 5044:5044 -p 5000:5000 -p 9300:9300 -it -v $(pwd):/home/$USER/ --name elk-5.0  sebp/elk:es500_l500_k500
+
+# set system vm.max_map_count to the minimal value acceptable by ES 5.0
+echo "increasing virtual memory for ES 5.0 ..."
+sudo sysctl -w vm.max_map_count=262144
+cp /etc/sysctl.conf /tmp/sysctl.conf
+echo "vm.max_map_count=262144" >> /tmp/sysctl.conf
+sudo cp /tmp/sysctl.conf /etc/sysctl.conf
```

Comment:
How many of these modifications are known necessary for this particular IT vs just things that have worked in the past? If it's possible to use a stock ES docker image, that'd be awesome. I'm guessing it's not possible or you would have used that originally :)

Are these still necessary if we restrict it to a super small data set?